core/src/main/java/org/apache/gravitino/tag/SupportsTagOperations.java

```diff
@@ -16,7 +16,6 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-
```

Comment:
Is this empty line expected?

Reply:
No, there's no rule that here should have an empty line.